src/common-lib/urls/urls.ts

```diff
@@ -135,6 +135,23 @@ export type PupilLessonLinkProps = {
   lessonSlug: string;
 };
 
+export type PupilLessonListingLinkProps = {
```

Comment:
I really don't understand the value of this file. We should ask Matt / Jim about this.

Reply:
its a single source of truth for all the urls; so if anything changes you just have to change it in the file